packages/terra-modal-manager/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Removed
+  * Removed the onMaximize and onMinimize defaulted implementation
```

Comment:
nitpick - can we say *why* we are removing these?

Reply:
updated here https://github.com/cerner/terra-framework/pull/1408/commits/7d7c2fda45f4cfa167f07897be9178a96fd3abc4